src/author.rs

```diff
@@ -1,18 +1,40 @@
+//! The `Author` struct and related utilities.
+//!
+//! This module defines the `Author` data structure together with its utility
+//! enum `AuthorParseError`.
+
+/// The author information.
 #[derive(Debug)]
 pub struct Author {
-    name: String,
+    /// The electronic contact information of the author.
```

Comment:
I think people know what an email is..

Reply:
A relic from the GPL 2.0; line 310 states "Also add information on how to contact you by electronic and paper mail."

And really, I encountered people who do not know what an email is.  Who think of it as some weird new stuff which might become obsolete soon.  So explaining it might not be the worst idea.

Reply:
Sure, there are such people, but they will not interact with this code base. So I think just calling it email is alright.

Reply:
Okay. :wrench:

Reply:
:wrench: :heavy_check_mark: